docs: :memo: add pages for current and previous decisions (#159)
seedcase-project/decisions: previous.qmd

---
title: Previous
listing:
  contents: why-*/*.qmd
  categories: true
  type: grid
  include:
    previous: true
  fields:
    - title
    - description
    - date
  sort: "date desc"
  max-items: 50
---

> These are decision posts we have previously made and followed but that
> no longer apply or are relevant

Reviewer: ```suggestion
These decision posts reflect previous decisions that are
no longer applicable or relevant.
```
